content/en/docs/languages/java/api.md

```diff
@@ -1,8 +1,1694 @@
-manualLinkTarget: _blank
-_build: { render: link }
-weight: 210
+title: Record Telemetry with API
+weight: 11
+aliases:
+  - /docs/languages/java/api-components/
+logBridgeWarning: >
+  While the `LoggerProvider` / `Logger` APIs are structurally similar to the
+  equivalent trace and metric APIs, they serve a different use case. As of now,
+  `LoggerProvider` / `Logger` and associated classes represent the [Log Bridge
+  API](/docs/specs/otel/logs/api/), which exists to write log appenders to
+  bridge logs recorded through other log APIs / frameworks into OpenTelemetry.
+  They are not intended for end user use as a replacement for Log4j / SLF4J /
+  Logback / etc.
+cSpell:ignore: Dotel kotlint Logback updowncounter
 ---
+
+<!-- markdownlint-disable blanks-around-fences -->
+<?code-excerpt path-base="examples/java/api"?>
+
+The API is a set of classes and interfaces for recording telemetry across key
+observability signals. The [SDK](../sdk/) is the built-in reference
+implementation of the API, [configured](../configuration/) to process and export
+telemetry. This page is a conceptual overview of the API, including
+descriptions, links to relevant Javadocs, artifact coordinates, and sample API
+usage.
+
+The API consists of the following top-level components:
+
+- [Context](#context-api): A standalone API for propagating context throughout
+  an application and across application boundaries, including trace context and
+  baggage.
+- [TracerProvider](#tracerprovider): The API entry point for traces.
+- [MeterProvider](#meterprovider): The API entry point for metrics.
+- [LoggerProvider](#loggerprovider): The API entry point for logs.
+- [OpenTelemetry](#opentelemetry): A holder for top-level API components (i.e.
+  `TracerProvider`, `MeterProvider`, `LoggerProvider`, `ContextPropagators`)
+  which is convenient to pass to instrumentation.
+
+The API is designed to support multiple implementations. Two implementations are
+provided by OpenTelemetry:
+
+- [SDK](../sdk/) reference implementation. This is the right choice for most
+  users.
+- [Noop](#noop-implementation) implementation. A minimalist, zero-dependency
+  implementation for instrumentations to use by default when the user doesn't
+  install an instance.
+
+The API is designed to be taken as a direct dependency by libraries, frameworks,
+and application owners. It comes with
+[strong backwards compatibility guarantees](https://github.com/open-telemetry/opentelemetry-java/blob/main/VERSIONING.md#compatibility-requirements),
+zero transitive dependencies, and
+[supports Java 8+](https://github.com/open-telemetry/opentelemetry-java/blob/main/VERSIONING.md#language-version-compatibility).
+Libraries and frameworks should depend only on the API and only call methods
+from the API, and instruct applications / end users to add a dependency on the
+SDK and install a configured instance.
+
+{{% alert %}} For an exhuastive javadoc reference of all OpenTelemetry Java
```

Comment:
How about just "the" instead of "an exhuastive"?

```suggestion
{{% alert title=Javadoc %}} For the Javadoc reference of all OpenTelemetry Java
```